components/ColonyQuestions.js

```diff
@@ -41,10 +38,8 @@ export default function ColonyQuestions({ currentColony }) {
     <FormContainer>
       <GeneralNotes currentColony={currentColony} />
 
-      <StyledQuestionCard>
-        <StyledQuestionLabel htmlFor="question1">
-          1. Stifte?
-        </StyledQuestionLabel>
+      <StyledCard>
+        <StyledLabel htmlFor="question1">1. Stifte?</StyledLabel>
```

Comment:
I think that it's not semantic to use a `label` as the heading of the checkboxes. What about the `fieldset` element ([see on mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox))?